html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+    <form action="" method="get" class="form">
+      <div class="form">
+        <label for="email">Email</label>
+        <br />
+        <input id="email" type="text">
+        </div>
+      <div>
+        <label for="why is coding cool">Why is coding cool?</label>
```

Comment:
The value of a `for` attribute should correlate to the value of the `id` attribute on the input it is correlated to. Additionally, neither `for` or `id` attributes can have values with spaces in them, please change this to kebab-case

Reply:
These are part of two different sections. The` id="email"` has a label above it with the corresponding `for` value